packages/components/src/components/combobox/combobox.custom.styles.ts

```diff
@@ -0,0 +1,34 @@
+import { css } from 'lit';
+
+export default css`
+  /** 
+   * Hide the default slot, because the filtered options are rendered in the listbox__options
+   */
+  .combobox__listbox slot:not([name]) {
+    display: none;
+  }
+
+  /**
+   * The filtered options
+   */
+  .listbox__options syn-option mark {
```

Comment:
There is no `mark` tag used in the component? Is this from the custom optionrenderer you are using? If so, we may have to defined this outside of the component.

Reply:
Yes, this comes if someone wants to highlight the characters with a `mark` element. 
The alternative would be, that everyone has to add this styling in their `getOption` render method, if they would like to have our recommended styling for highlighted elements. 

They would need to do this in the `getOption`:
![image](https://github.com/user-attachments/assets/fc631315-5bd9-4169-9768-3da6be6a3288)

And then the styling is still not exactly correct, because this styling would be missing: 
![image](https://github.com/user-attachments/assets/31e0094b-47f9-42d9-9b22-d95ed4213206)

and the result would then look like this for keyboard highlighting of an option (the `g` should be white also):
![image](https://github.com/user-attachments/assets/c75c760e-2953-412a-b241-ed6ec3daabc2)

For slotted elements we also sometimes do it for the people, to style the elements (e.g. Slotted syn-icon) so they don't have to do it.

Therefore I would let the styling stay there

Reply:
I am fine with this, as long as we document it properly (e.g. something like: "when using `getOption`, make sure to use a `<mark>` element for brand compliance")

Reply:
Where would you add the documentation? To the property documentation ( in the table at the top) or at the highlight story? 
Both documentations are already relatively long😅

Reply:
I think I would add it to the documentation for `getOption`. Maybe we should also think about to alter `getOption`s signature, maybe like:

```typescript
// Current
getOption(option: SynOption, queryString: string)`

// New:
getOption(option: SynOption, queryString: string, autoMark?: boolean = true)`

// or even:
type GetOptionOptions = {
  autoMark?: boolean;
};

getOption(
  option: SynOption,
  queryString: string,
  opts?: GetOptionOptions = { autoMark: true, }
)`
```

We could then go the route of defining our default like so:

```typescript
@property() getOption: (option: SynOption, queryString: string, autoMark?: boolean = true) => TemplateResult | string | HTMLElement = (option, q, autoMark) => autoMark ? YOUR_CUSTOM_LOGIC_CURRENTLY_IN_TEMPLATE : option;
```

Reply:
If we will integrate the highlighting function directly in the combobox, maybe it would make sense to add a boolean property for it. 
So the people just need to do something like this: 

```html
<syn-combobox highlight ></syn-combobox>
```

if we do it as option in the getOption function, they would need to go into javascript

```js
const combobox = document.querySelector('syn-combobox');
const oldGetOption = combobox.getOption;
combobox.getOption = (option, query, opts)=> oldGetOption(option, query, { ... opts, autoMark: true } );
```

Reply:
Yeah, this was probably not well thought through. We could also go another route and export two getOption functions, one with and one without highlighting. However, this would also require setting it on every instance, which is unfortunate.

Reply:
Just a thought from the outside (😂): I think it's totally fine to set specialiced options on every instance. As soon as you alter things, there will be many things involved anyways. You could export a "mark" method as well that could be used then on the content.

The most important thing might be a good default and then I believe it's totally fine to give some effort to the devs.

Reply:
Added a highlight option render utility function, which can be imported and used

Reply:
done 👍